src/platform/input/constants.js

```diff
@@ -52,6 +52,13 @@ export const EVENT_MOUSEUP = 'mouseup';
  */
 export const EVENT_MOUSEWHEEL = 'mousewheel';
 
+/**
+ * Name of event fired when the mouse moves out or enters another element.
```

Comment:
```suggestion
 * Name of event fired when the mouse moves out or enters another DOM element.
```
Making it clear that we aren't talking about pc Elements